cypresse2e/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "trailingComma": "all",
+  "tabWidth": 2,
+  "semi": true,
+  "singleQuote": true
```

Comment:
Ohh one more thing, I am sorry @galeza, but can we also change it to single quote: false, and re run prettier, so that way we keep it consistence across the frontend and the testing service... Or do you think we should keep it as single quotes?

Reply:
@idjevm 
I set single quote: false and ran prettier.
If ok, please merge.